### DISCUSSION TITLE
How we can change the current admin ui styles?

### DISCUSSION THREAD

**Discussion by muhad-bk** [keystonejs/keystone#6630] [Event Q&A Q3 2021]
How to change the current admin UI styles?  
Is it possible to override the styles?

**Comment by raveling** [comment]
@muhad-bk 

AdminUI theming is on the roadmap, but at the moment there is no recommended way to change admin ui styles for existing pages.

You can, however add custom [pages](https://keystonejs.com/docs/guides/custom-admin-ui-pages), custom [nav](https://keystonejs.com/docs/guides/custom-admin-ui-navigation), and your own [logo](https://keystonejs.com/docs/guides/custom-admin-ui-logo).

Also, FYI – we've closed this category as the [community Q&A event](https://youtu.be/r1IJh-iHm1c) happened last week.

For the time being, please post your support questions to our [community Slack](https://community.keystonejs.com/). If we run another event like the last one we'll create a new discussion category.
